packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -249,4 +277,15 @@ contract GoldToken is
     emit Transfer(msg.sender, to, value);
     return true;
   }
+
+  /**
+   * @notice internal CELO transfer from one address to another.
+   * @param to The address to transfer CELO to. Zero address will revert.
+   * @param value The amount of CELO to transfer.
+   * @return True if the transaction succeeds.
+   */
+  function _transferWithCheck(address to, uint256 value) internal returns (bool) {
+    require(to != address(0), "transfer attempted to reserved address 0x0");
```

Comment:
Do we know what address 0x0 was reserved for?

Reply:
limiting the ERC20 transfer is not actually very useful because you can do a native transfer instead.

The 0x0 address was limited because it's common for UIs to be buggy and using zero as a empty state, then burning the user value. This obviously won't happen for the dead address.